tests/test_cli/test_registry/test_utils.py

```diff
@@ -277,7 +277,7 @@ def test_extract_wrong_file_type(self):
         """Test for extract method wrong file type."""
         source = "file.wrong"
         target = "target-folder"
-        with self.assertRaises(Exception):
+        with self.assertRaises(ValueError):
```

Comment:
Same question

Reply:
Pylint informed that this check was too generic. I could skip the failure, but it seemed very easy to solve. Would you prefer to revert this change and skip the lint error?

Reply:
If the test pass no need to revert the change